packages/ai/src/ai/web/endpoints/webhook.py

```
@@ -0,0 +1,250 @@
+            _active_tasks[token] = {
+                'pipeline_id': pipeline_id,
+                'status': 'accepted',
+                'created_at': now.isoformat(),
+                'input_data': body,
+                '_monotonic_created': time.monotonic(),
+            }
+
+            # Dispatch the pipeline execution as a background task so the
+            # webhook caller receives a response immediately and can poll
+            # /webhook/{pipeline_id}/status/{token} for progress.
+            asyncio.create_task(_execute_pipeline(token, pipeline_id, body))
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)